MovieGPT-api/src/main/java/lt/techin/group/project/model/Media.java

```diff
@@ -0,0 +1,33 @@
+package lt.techin.group.project.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lt.techin.group.project.rest.dto.MediaDto;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Media {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+    private String title;
+    private String description;
+    private String imageUrl;
+    private String thumbnailUrl;
+    private Integer releaseYear;
+    @ManyToOne(fetch = FetchType.EAGER)
+    private Genre genre;
```

Comment:
Replace with:
```
    @ManyToMany(fetch = FetchType.EAGER)
    private Set<Genre> genre;
```

Reply:
Solved, ManyToMany introduced (wasnt working before due to Lombok Data annotation)